listenbrainz_spark/recommendations/train_models.py

```diff
@@ -341,31 +345,34 @@ def main(ranks=None, lambdas=None, iterations=None, alpha=None):
 
     # Rdds that are used in model training iterative process are cached to improve performance.
     # Caching large files may cause Out of Memory exception.
-    training_data.persist()
-    validation_data.persist()
+    # commenting out till we are sure that the cached data isn't causing RPC errors.
```

Comment:
let's remove the lines, it's easy to get them back from git, and it's generally very easy to forget about things like this.